internal/engine/image_build_and_deployer.go

```diff
@@ -36,6 +36,10 @@ func NewImageBuildAndDeployer(b build.ImageBuilder, k8sClient k8s.Client, env k8
 func (ibd *ImageBuildAndDeployer) BuildAndDeploy(ctx context.Context, manifest model.Manifest, state BuildState) (br BuildResult, err error) {
 	span, ctx := opentracing.StartSpanFromContext(ctx, "daemon-ImageBuildAndDeployer-BuildAndDeploy")
 	defer span.Finish()
+	defer func() {
+		span.Finish()
+		output.Get(ctx).Printf("") // space between Pipelines
```

Comment:
nbd but this is a weird logical grouping of things. i'd put the extra printline below, at ln 56 along with the `EndPipeline` call (or better yet, just put that logic in the `EndPipeline` function itself)